client/data/marketplace/utils.ts

```diff
@@ -18,5 +18,5 @@ export const getPluginsListKey = (
 		options.tag && ! options.searchTerm ? options.tag : '',
 	];
 
-	return [ PLUGINS_CACHE_VERSION.toString(), key, ...keyParams ];
+	return [ PLUGINS_CACHE_VERSION.toString(), ...key, ...keyParams ];
```

Comment:
I know this was here before, but I wonder if the `key` part should be at the beginning and the cache version would be afterwards.

Reply:
Good point, I'll double-check and add a commit.

Reply:
Sounds good. Can be done in another PR if you don't want to clutter this one.

Reply:
See https://github.com/Automattic/wp-calypso/pull/74720/commits/d8ddad71b60b687e64b1dc92f1d7f2ecdabc30c0